### ISSUE TITLE
[Feature Request] Linux ARM64 support

### ISSUE THREAD

**Issue by itlancer** [airsdk/Adobe-Runtime-Support#1884]
## Feature Description
AIR should support Linux for ARM64 architecture.
It will allows a huge number of developers uses AIR for mass-market IoT devices like Raspberry Pi and so on.
As a first step Debian and Raspbian should be supported I think.

Previously AIR already returned support for Linux x86_64 https://github.com/airsdk/Adobe-Runtime-Support/issues/94. Please go further.

## Known Workarounds
none

**Comment by dmunsie** [comment]
"It will allows a huge number of developers uses AIR for mass-market IoT devices like Raspberry Pi and so on." 

Really, mass market? Could you explain the monetization possibilities to substantiate this request? You have my curiosity... .thanks.

**Comment by itlancer** [comment]
@dmunsie 
I can tell about what our customers want. Crossplatform applications with multimedia capabilities.
Most of consumer TV boxes uses Linux for ARM or Android. I'm not mention tvOS here cause it has different business model. Almost all telecom operators who provide TV services nowadays provide TV boxes based on Linux ARM or Android.

IoT devices (smart home an so on). Raspberry Pi, Orange Pi, Khadas, NVIDIA Jetson Nano - just examples for developing such solutions. Only part of them support both Linux ARM and Android. But all of them support Linux ARM.

B2B solutions: retail price checkers, POS terminal with screens, meeting room booking tablets, video streamers systems. Almost all of them based only on Linux ARM.

Industrial and OEM: factory tablets/screens, car multimedia systems, ARM servers.


My opinion: more capabilities from AIR framework could lead more usages (device units and so on) of final AIR applications. More installations and more projects lead to requst of customers for more features. More features and more installations leads to more AIR developers needed. So more AIR licenses could be sell.

Also Linux ARM support could be useful for education purposes too. Universities and schools could be interested in AIR as one of technology which could be used as a instrument for creation education projects for Linux ARM devices. That what @ajwfrost  also mention about AIR further promotion  https://github.com/airsdk/Adobe-Runtime-Support/discussions/1465#discussioncomment-1834607

**Comment by LilithRainbows** [comment]
> 

Agree, nowadays linux is used a lot on arm64 devices, a version of air for that architecture would be very welcome.

**Comment by pol2095** [comment]
In [AIR 33.1.1.821 Release Notes](https://airsdk.harman.com/download/33.1.1.821)
`Currently only x86_64 support – ARM64 is planned and potentially 32-bit variants if needed`

**Comment by itlancer** [comment]
It supported and pretty mature. Tested with AIR 51.3.3.1. Thanks!
But still have some issues like https://github.com/airsdk/Adobe-Runtime-Support/issues/4273.
